ogr/abstract.py

```diff
@@ -1875,6 +1878,19 @@ def get_commit_comments(self, commit: str) -> list[CommitComment]:
         """
         raise NotImplementedError()
 
+    def get_commit_comment(self, commit_sha: str, comment_id: int) -> CommitComment:
+        """
+        Get commit comment.
+
+        Args:
+            commit_sha: SHA of the commit
+            comment_id: ID of the commit comment
+
+        Returns:
+            Object representing the commit comment.
+        """
+        raise NotImplementedError()
```

Comment:
ya… this should be an abstract class, it would've been much easier to see unimplemented methods and missing functionality (e.g., on Pagure' side), but that sounds more like a Friday thing than including this in here…